solr/CHANGES.txt

```diff
@@ -57,7 +57,7 @@ Improvements
 
 Optimizations
 ---------------------
-(No changes)
+* SOLR-8306: Enhance ExpandComponent to allow expand.hits=0 (Marshall Sanders, Amelia Henderson)
```

Comment:
Maybe change the wording here, something like `Do not collect expand documents when expand.rows=0`
there is no `expand.hits` parameter

Reply:
Thanks for that. updated